src/main/java/org/fcrepo/client/FedoraTypes.java

```diff
@@ -24,6 +24,18 @@
  */
 public class FedoraTypes {
 
+    // Type representing an LDP resource
+    public static final String LDP_RESOURCE = "http://www.w3.org/ns/ldp#Resource";
+
+    // Type representing an LDP non-RDF source/binary
+    public static final String LDP_NON_RDF_SOURCE = "http://www.w3.org/ns/ldp#NonRDFSource";
+
+    // Type representing an LDP direct container
+    public static final String LDP_DIRECT_CONTAINER = "http://www.w3.org/ns/ldp#DirectContainer";
```

Comment:
Should `LDP_BASIC_CONTAINER` be included as well?

Reply:
Oops, yes, I had meant to add it then proceeded not to :)